app/controllers/admin/organisations_controller.rb

```diff
@@ -1,5 +1,26 @@
 class Admin::OrganisationsController < AdminController
+  helper_method :sort_column, :sort_direction
+  before_action :index, :authorise_admin
+
   def index
-    @organisations = Organisation.order(:name).all
+    @organisations = Organisation.order("#{sort_column} #{sort_direction}").all
+  end
+
+private
+
+  def authorise_admin
+    redirect_to root_path unless current_user.admin?
+  end
```

Comment:
Does the AdminController which we inherit from here do this authorisation already?